### ISSUE TITLE
Jspm link hangs. I never get 'ok   Package linked.'

### ISSUE THREAD

**Issue by marthinus-engelbrecht** [jspm/jspm-cli#1931]
Hi,

When attempting to link a package the link command never exits.  But it does seem to link the package because `jspm install --link npm:package@version` works. 

Something I've noticed that might be related is that it adds the package to the main project's map in the config.js but it doesn't seem to add the child mappings in the config.js. e.g 

```js
"npm:package@version" : {
   "dependencyA" : "npm:dependencyA@version"
}
```

**Comment by guybedford** [comment]
It's probably trying to process all of your node_modules folder or something like that. You can try adding `ignore: ['node_modules']` to the package.json. I believe this issue was fixed in jspm 0.17.

**Comment by marthinus-engelbrecht** [comment]
Will give it a try. Thanks
